entrypoint.sh

```diff
@@ -15,9 +29,9 @@ else
 fi
 
 if [[ -n "$INPUT_TFSEC_EXCLUDE" ]]; then
-  TFSEC_OUTPUT=$(/go/bin/tfsec ${TFSEC_WORKING_DIR} --no-colour -e "${INPUT_TFSEC_EXCLUDE}")
+  TFSEC_OUTPUT=$(/go/bin/tfsec ${TFSEC_WORKING_DIR} --no-colour -e "${INPUT_TFSEC_EXCLUDE} ${TFSEC_FORMAT} ${TFSEC_FILE}")
```

Comment:
What about having only one param for those two parameters here. What I'm afraid of is that someone gives the `TFSEC_FILE` but not the `TFSEC_FORMAT` which might result in an error or unwanted result. We could create this new variable in the check you do before if there is a format or not. WDYT ?

Reply:
i agree it needs to be different, i still think new params though

Reply:
TFSEC_FORMAT always has a default value so it cant happen that you have TFSEC_FILE and not TFSEC_FORMAT? I think it may suit someone to provide the format but not the file? anyway thats my intention?

Reply:
So I've played a bit with it and from what I can see:
- if the `--out` is defined without a format different than `default`, it will just print it
- a default format is defined
- this is a bit confusing to use a parameter that is not working when the default format is used (i.e: `tfsec --out test.txt --format default`)

Now, this is how tfsec works, I guess this has to be the expected behavior even if I don't really agree with the way it works. 

For the default format value, we can keep it like that but I'd rather try to rely on the tool defaults than forcing a default in here. What I mean is if tomorrow the tool decides to change the default, we won't be up to date with this decision as we hardcode a value here.

Does this make sense to you?

Reply:
LGTM